### DISCUSSION TITLE
Count of issues on my repositories seems off

### DISCUSSION THREAD

**Discussion by Nixinova** [lowlighter/metrics#468] [Q&A] [answered]
From [this search query](https://github.com/issues?q=is%3Aissue+user%3ANixinova+is%3Aopen) you can see I have 0 open / 12 closed issues on Nixinova/* repos.

However the metrics list 1 open / 23 closed:

![image](https://user-images.githubusercontent.com/42429413/128956760-c1e524bc-b18b-45ff-bb88-672c14b7808c.png)

Now I thought this may be because I have open issues on repositories I am an owner of, but this isn't the case; [this query](https://github.com/issues?q=is%3Aissue+user%3ANixinova+org%3ANovasheets+org%3ACarbonchess+is%3Aopen+) gives 3 open / 28 closed, which also doesn't match.

Where is the 1 / 23 figure coming from?

**Comment by lowlighter** [comment]
This is computed by the `core` plugin rather than the `followup` one (it was there before the split into plugins):
https://github.com/lowlighter/metrics/blob/45d86dd00ec760237ad9b32bd4e40606d3cf33f1/source/plugins/core/index.mjs#L65-L69

Since `data.user.repositories.nodes` depends on `repositories_affiliations` options which change which repositories are actually fetched, I assume it's taking into account some repositories that you may not be owner of but instead contributed too 🤔 ?

Either way, with the current wording, this plugin should indeed fetch only the repositories where user is owner/creator (or eventually with write access).

We probably need decouple this plugin from the `core` one to reflect more accurate stats when using broad affiliations

**Comment by lowlighter** [comment, marked as answer]
Since this plugin has been somewhat refactored in the meantime, I'll archive this discussion 👍 
Using `indepth` mode should yield better results
